examples/functions/newModes/IXSI Service 1 - Static Data.xml

```diff
@@ -113,7 +113,7 @@
 							<proposedByServices>
 								<OnlineServiceRef version="any" ref="GE:OnlineServiceRef:12"/>
 							</proposedByServices>
-							<VehicleSharingOperationRef version="any" ref="GE:VehicleSharingOperation:12"/> <!--Mandatory-->
+							<VehicleSharingRef version="any" ref="GE:VehicleSharingOperation:12"/> <!--Mandatory-->
```

Comment:
I don't think it should refer to an VehicleSharingOperation if it is not an VehicleSharingOperation itself, and it it would be than it would require something like nameOfRefClass.

Reply:
changed